main_routers/system_router.py

```
@@ -0,0 +1,488 @@
+        response = await client.chat.completions.create(
+            model=model,
+            messages=[
+                {"role": "system", "content": emotion_analysis_prompt},
+                {"role": "user", "content": text}
+            ],
+            extra_body=extra_body if extra_body else None
+        )
+        
+        emotion = response.choices[0].message.content.strip()
+        
+        return {"success": True, "emotion": emotion}
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)